### PR TITLE
Restructure standup around stories

### PR DESCRIPTION
**Pull request by JimMadge**
Update weekly standup template to focus reporting on stories rather than individuals.

Supersedes #61

### CODE REVIEW

**Comment by Arielle-Bennett** [comment]
Is this going to be a weekly thing? And will the groups who are "accountable" be the ones responsible for completing each section or someone else?

File: weekly-standup-template.md [hunk 1]
```diff
@@ -17,33 +17,309 @@ colorlinks: true
...
 
-Group pieces of work into stories.
+Add your name to any story you have worked on and include a rough estimate of the time spent.
+
+Add pieces of work into stories to the relevant stories.
+Add some brief context to explain what the work does and why we did it.
 Please include links.
 
-Please indicate any work that is new, or otherwise not included on the project board.
+Work that not related to a story can go in the 'Other' section.
```
Comment (Davsarper) [inline weekly-standup-template.md:35]:
```suggestion
Work that is not related to a story can go in the 'Other' section.
```

**Review by Davsarper — APPROVED** [review]
All good, thanks for putting it together.
I like the reporting template focus on linking issues, PRs etc. and I would keep it as is, but keep in mind part of the conversation last week was "not everything will be an issue". Still is just an exampe and I like that where there is something to link it encourages to do so

**Comment by JimMadge** [comment]
> I like the reporting template focus on linking issues, PRs etc. and I would keep it as is, but keep in mind part of the conversation last week was "not everything will be an issue". Still is just an exampe and I like that where there is something to link it encourages to do so

That's true. The document is quite long in this format so I think there is a balance between keeping it brief and making it useful. I hope that the prompts that are there give the correct impression.

**Pull request merged**